Lib/test/test_compiler_assemble.py

```diff
@@ -70,3 +70,40 @@ def test_simple_expr(self):
         ]
         expected = {(3, 4) : 3.5, (-100, 200) : 50, (10, 18) : 14}
         self.assemble_test(insts, metadata, expected)
+
+
+    def test_expression_with_pseudo_instruction_load_closure(self):
+
+        def mod_two(x):
+            def inner():
+                return x
+            return inner() % 2
+
+        inner_code = mod_two.__code__.co_consts[1]
```

Comment:
Add an assertion that inner_code is a code object instance (it will be easier to debug if something changes and we need a different index into co_consts).